modules/ROOT/examples/live-demos/autocorrect/index.js

```diff
@@ -0,0 +1,6 @@
+tinymce.init({
+  selector: 'textarea#autocorrect',
+  plugins: 'autocorrect',
+  autocorrect_autocorrect: true,
```

Comment:
Is the difference between here and the comment left here: https://github.com/tinymce/tinymce-docs/pull/2500#discussion_r964254746, that we do include default settings for the specialised demos?

Reply:
I removed the default for autocorrect but keep `autocorrect_capitalize` since it is false by default.

Reply:
That's not what your documentation says: https://github.com/tinymce/tinymce-docs/blob/feature/6/DOC-1753/modules/ROOT/partials/configuration/autocorrect_capitalize.adoc

Reply:
I should add if the above is wrong, then we probably should add it back to the full featured demos.

Reply:
> That's not what your documentation says: https://github.com/tinymce/tinymce-docs/blob/feature/6/DOC-1753/modules/ROOT/partials/configuration/autocorrect_capitalize.adoc

It is fixed now

Reply:
> I should add if the above is wrong, then we probably should add it back to the full featured demos.

Do you mean that capitalize should be enabled by default for full feature demo?

Reply:
Yep, it's supposed to be full featured and have as much enabled as possible/makes sense.